homeassistant/components/hassio/handler.py

```diff
@@ -590,14 +590,25 @@ async def send_command(
                 timeout=aiohttp.ClientTimeout(total=timeout),
             )
 
-            if request.status != HTTPStatus.OK:
-                _LOGGER.error("%s return code %d", command, request.status)
+            if response.status == HTTPStatus.BAD_REQUEST:
```

Comment:
```suggestion
            if response.status != HTTPStatus.OK:
```
This handling isn't specific to `BAD_REQUEST`. Supervisor returns json with a message for all errors: <https://github.com/home-assistant/supervisor/blob/main/supervisor/api/utils.py#L119-L142>. We should have the same handling anytime the response is not `HTTPStatus.OK`

Reply:
Right it seems that even `APIForbidden` which returns HTTP status code 403 usually has a message. Make sense to transfer this into the `HassioAPIError` exception as well :+1: